src/viewer/Entity.tsx

```diff
@@ -63,11 +63,22 @@ class Entity extends React.Component<Props & CardsProps & PortalProps, State> {
 		if (!prevState.isHovering && this.state.isHovering) {
 			this.statisticsTimeout = window.setTimeout(() => {
 				this.statisticsTimeout = null;
-				this.setState({ showStatistics: true });
+				this.setState({ isMeaningfulHover: true });
 			}, 500);
 		}
 		if (prevState.isHovering && !this.state.isHovering) {
-			this.setState({ showStatistics: false });
+			this.setState({ isMeaningfulHover: false });
+		}
+		if (
+			this.state.isHovering &&
+			!prevState.isMeaningfulHover &&
+			this.state.isMeaningfulHover
+		) {
+			if (this.props.dbfId) {
+				ga("send", "event", "Hover", "Card", this.props.dbfId.toString());
```

Comment:
I suggest casting strings this way:
```suggestion
				ga("send", "event", "Hover", "Card", "" + this.props.dbfId);
```

You can also use `String(this.props.dbfId)`, as suggested in the [Airbnb style guide](https://github.com/airbnb/javascript#coercion--strings) (a great doc to read!).

Reply:
great, I'll read it, thanks!